zip-draft-XXXX.rst

```diff
@@ -0,0 +1,289 @@
+   design should ensure compliance with applicable laws and regulations to
+   support the long-term sustainability of the funding model.
+
+4. **Efficient Allocation Preparation**: The lockbox must be prepared to
+   allocate resources efficiently once the disbursement mechanism is defined.
+   This includes ensuring that funds are readily available for future projects
+   and not tied up in organizational overhead.
+
+5. **Potential Tax Efficiency**: The lockbox should be designed with the
+   potential to defer taxable events until the liquidation of funds by
+   recipients, allowing for more flexibility in handling tax liabilities
+   compared to direct funding of US-based organizations.
+
+6. **Security of Funds**: The lockbox must implement robust security measures
+   to protect against unauthorized access or misuse of funds. This includes
+   ensuring the integrity of the future decentralized voting/signature
+   mechanism.
+
+7. **Future Adaptability**: The lockbox design should be flexible and adaptable
+   to accommodate the disbursement mechanism defined in a future ZIP. This
+   ensures that the system can evolve based on feedback and new developments in
+   the Zcash ecosystem.
+
+
+Non-requirements
+================
+
+This ZIP specifically addresses the creation of an in-protocol "lockbox" for
+accumulating a portion of the Zcash block reward post-November 2024 halving.
+The following aspects are explicitly deferred to future ZIPs and are not
+covered by this proposal:
+
+1. **Disbursement Mechanism**: The exact method for disbursing the accumulated
+   funds from the lockbox is not defined in this ZIP. The design,
+   implementation, and governance of the disbursement mechanism will be
+   addressed in a future ZIP. This includes specifics on how funds will be
+   allocated, the voting or decision-making process, and the structure of the
+   decentralized mechanism (such as a DAO).
+
+2. **Specific Allocation Percentages**: While this ZIP proposes the allocation
+   of |percentage| of the block reward to the lockbox, it does not prescribe
+   the detailed percentages for various funding categories or the specific
+   amounts that will be distributed to different projects or entities.
+
+3. **Regulatory Compliance Details**: The proposal outlines the potential to
+   reduce regulatory risks by avoiding direct funding of US-based
+   organizations, but it does not detail specific regulatory compliance
+   strategies. Future ZIPs will need to address how the disbursement mechanism
+   complies with applicable laws and regulations.
+
+4. **Technical Implementation**: The technical specifics of how the lockbox
+   will be integrated into the Zcash protocol, including any necessary changes
+   to the codebase and the precise methods for ensuring security and
+   transparency, are not covered in this ZIP. These details will be provided in
+   subsequent proposals and technical documentation.
```

Comment:
I don't think it's possible to leave this out - if the ZIP is approved it will have to be implemented and we'll need this information regardless. But note that ZIPs don't have to explain how things are going to be implemented, just what is the change in the protocol that needs to be implemented (i.e. what you have already written in the Specification section, so my suggestion is to simply remove this item)